libs/components/src/lib/Accordion/index.tsx

```diff
@@ -0,0 +1,72 @@
+import cls from "classnames";
+import { FC, ReactNode, useState } from "react";
+import { PlusIcon } from "../Icons/plus";
+import classes from "./index.module.scss";
+import { useData } from "./index.hook";
+
+interface AccordionItemProps {
+  title: string;
+  children: ReactNode;
+}
+
+const AccordionItem: FC<AccordionItemProps> = ({ title, children }) => {
```

Comment:
please use shadow rendering for AccordionItem, In each index file we should have just one export.